spec/fabricators/queue_item_fabricator.rb

```diff
@@ -0,0 +1,3 @@
+Fabricator(:queue_item) do
+  
```

Comment:
I actually just meant line 2 is unnecessary white space. Personally I don't have a problem with leaving the method on two lines, but you definitely shouldn't have third blank line as that is rather universally bad.